_posts/2021-01-25-12.0.0.Final.adoc

```diff
@@ -0,0 +1,202 @@
+* Cross site improvements:
+** Access to remote cluster k8s API is no longer mandatory.
+** Manual configuration.
+** Namespaces can be different.
+** Multinamespace and global installation mode.
+* TLS can be disabled via CR configuration, even in environments where certificate management is built-in (e.g. OpenShift).
+* Support for the Quarkus native Infinispan image.
+* Anti-affinity configuration.
+
+== Documentation
+
+* Documentation for configuring cluster transport was overhauled in Infinispan 12. We made numerous changes, based on direct feedback from the community, to our JGroups content. We also added details on how to configure SYM_ENCRYPT and ASYM_ENCRYPT protocols, which were previously available only in the JGroups documentation set. 
+* You can find new chapters on Configuring Cluster Transport in the Server Guide and Embedding Infinispan Guide. The details for JGroups encryption are also available in the guide for Securing Infinispan.
+* Off-Heap configuration has often been misunderstood and we’ve had several questions about what it actually means to store data in native memory outside the JVM heap. To address these questions, and spell out some of the benefits and potential downsides of using off-heap storage, we’ve added some conceptual information to our Configuration Guide.
+* Our documentation on Configuring Infinispan to Marshall Java Objects is also updated in an effort to improve clarity and be more task-oriented.
+* Along with all the refactoring and improvements that have been made to the Search API, we’ve made some improvements to our documentation for Querying Values in Infinispan Caches.
+* Lastly, we noticed that the documentation for Clustered Locks was a little out of date and potentially confusing so we spent time to rework that content to make sure it’s accurate.
+
+As always, the Infinispan team hope you find the documentation useful and complete. We’d love to hear from you and
+really value feedback from our community. If you think something is missing from the docs or spot a correction,
+please get in touch and we’ll get on it straight away.
+
+== Other
+
+* We have decided to adopt https://github.com/conscious-lang/conscious-lang-docs[Conscious language] in Infinispan, both in our code and in our documentation. For example, Infinispan's way of configuring which classes are allowed to be marshalled/unmarshalled is now called _allow list_. If you find other places where we are using inappropriate terms, please don't hesitate to contact us.
+* We’ve added some UI writing guidelines and Infinispan terminology to our Contributor’s Guide in an effort to create a consistent user experience.
+* As you may have noticed, our website has gone through some extensive redesign.
+
+
+== Release notes and upgrading
+
+You can look at the detailed https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12345018[release notes] to see what has changed since CR1.
```

Comment:
for some reason, I don't have anonymous access to the release notes. It asks me to login. 
is there a way to enable anonymous access?

Reply:
I don't think there is.